packages/node-cli/src/commands/setup/SetupInputHandler.js

```diff
@@ -63,11 +63,11 @@ module.exports = class SetupInputHandler extends BaseInputHandler {
 		let authIdAnswer;
 		const choices = [];
 
-		const authIDList  = await getAuthIds(this._sdkPath);
-		if (authIDList.status === ActionResult.STATUS.ERROR) {
-			throw authIDList.errorMessages;
+		const authIDMap  = await getAuthIds(this._sdkPath);
```

Comment:
This const is actually an actionResult. The map of authIDs is going to be the data. I think this is the bad naming that leads what I think is an error in  line 160
`(fieldValue) => validateAuthIDNotInList(fieldValue, Object.keys(authIDMap))`
Maybe I'm wrong, but can you check it.